c10/core/Storage.h

```diff
@@ -53,6 +32,23 @@ struct C10_API Storage {
             allocator,
             resizable)) {}
 
+  static Storage create_legacy(at::DeviceType device_type) {
```

Comment:
can we just kill this one?  It's only called from one place, and that place can just do this.